Tools/CISupport/ews-build/steps.py

```diff
@@ -2596,8 +2596,9 @@ def getQueueStatusFromList(self, change_id, pr_number):
         for queue in queues_for_safe_merge:
             queue_data = response.json().get(queue, None)
             # jsc-arm7-tests will not set its status if skipped, so we condition on jsc-armv7
-            if queue == 'jsc-armv7-tests' and response.json().get('jsc-armv7', None).get('state', None) == 3:
```

Comment:
Another alternative is:
if queue == 'jsc-armv7-tests' and response.json().get('jsc-armv7', {}).get('state', None) == 3:

Also, it would be good to add a check earlier to ensure response is not None and response.json() is not None, and maybe exit early (or retry) if so.